dlt/helpers/airflow_helper.py

```diff
@@ -135,7 +135,7 @@ def add_run(
         pipeline: Pipeline,
         data: Any,
         *,
-        decompose: Literal["none", "serialize"] = "none",
+        decompose: Literal["none", "serialize", "parallel"] = "none",
```

Comment:
The method `add_run()` looks very big. Maybe we should split it into several smaller methods?

Reply:
yes - I want to extract internal runner `def _run() -> None:` an let people use it to run any task. this is the second part of this ticket.

this will require refactoring of this function